### PR TITLE
use background field to calculate adiabatic profile

### PR DESCRIPTION
**Pull request by RanpengLi**
This pull request updates the result of an existing test. When having multiple components, we use the background field to calculate the adiabatic profile.

* [X] I have followed the [instructions for indenting my code](../blob/main/CONTRIBUTING.md#making-aspect-better).

### CODE REVIEW

File: source/adiabatic_conditions/compute_entropy_profile.cc [hunk 1]
```diff
@@ -74,19 +74,13 @@ namespace aspect
       in.requested_properties = MaterialModel::MaterialProperties::density | MaterialModel::MaterialProperties::additional_outputs;
       in.velocity[0] = Tensor <1,dim> ();
       // The entropy along an adiabat is constant (equals the surface entropy)
-      // TODO : need to make it work for more than one entropy field
+      // When there are more than one entropy field, we use the background field to compute adiabat profile
```
Comment (gassmoeller) [inline source/adiabatic_conditions/compute_entropy_profile.cc:77]:
```suggestion
      // When there is more than one entropy field, we use the background field to compute the adiabatic profile
```

**Review by gassmoeller — COMMENTED** [review]
Just a few typos. Ready otherwise.
Just for reference this is a fix for #5622.

**Comment by RanpengLi** [comment]
Thank you! I have fix them accordingly.

**Review by gassmoeller — APPROVED** [review]

**Pull request merged**